### DISCUSSION TITLE
first try, unsuccessful

### DISCUSSION THREAD

**Discussion by gpion** [mctinker/Map-Tasker#21] [General]
Hello

- I've exported my tasker configuration 
- I have run "maptasker -g -debug"
- selected the .xml file
- the windows close almost immediately but the python process is still running

I'm unable to locate the maptasker.log file (looks like it's not even created)

Running on Windows 10, up to date, Python 3.13.0, Tcl/Tk version  8.6.14

**Comment by mctinker** [comment]
First understand that Windows 10 is unsupported.  MapTasker has not been tested on this platform.
 
Secondly, the '-g' option overrides all other runtime options.  If using the GUI, you must then turn on debug within the GUI (Debug tab in GUI).
 
Finally, make certain you have installed the latest version of MapTasker, which is version 5.3.2 .
 
Let me know how you make out.

**Reply by gpion** [reply]
>  Windows 10 is unsupported. MapTasker has not been tested on this platform.

Oh sorry, since I've read in the notes that _"Support Windows 11"_ is on the Todo list I thought this implied that Windows 10 was OK.
So only Linux then ?

> make certain you have installed the latest version of MapTasker, which is version 5.3.2 .

According to "maptasker -v"  that's the installed version (just did it that morning for the 1st time ever)

> Secondly, the '-g' option overrides all other runtime options. If using the GUI, you must then turn on debug within the GUI (Debug tab in GUI).

I can't : with "-g" option the GUI automatically activate a file selector choose the tasker xml file and the interface immediately disappear after that (but the command stays running) although consuming no CPU at all according to the task manager.

**Comment by gpion** [comment]
Just tried maptasker -file _\<path to tasker export xml file\>_ and MapTasker.html is correctly generated.
I've now have to see if that can be useful for me.

**Comment by mctinker** [comment]
Ok, glad the browser feature is working for you.  You can key in 'maptasker -h' to see the other runtime options.
 
I am guessing that there is a problem either with Tcl/Tk or tkinter as part of Python 3.13.  You've encountered a "catch-22" situation in which you can not turn on debug for the GUI.  I'll look into possibly fixing this in the next version.
